README.md

```diff
@@ -532,8 +540,15 @@ public class InitAPIs implements InitAPI {
 
 ## Adding code to run periodically
 
+This API can be used if your module *instance* needs to perform ongoing tasks.
+Consider using Okapi's timer facility if the task to be performed is "per-tenant"
+and only needs to be executed on one instance and not all instances of the module.
+If the task performs operations on persistent storage, that is typically a sign
+that it should be using the Okapi timer facility (not was is presented in this section).
```

Comment:
Can you add a link to https://github.com/folio-org/okapi/blob/master/doc/guide.md#timer-interface ?

Reply:
sure